app/Views/welcome_message.php

```diff
@@ -150,6 +150,26 @@
         .further h2:first-of-type {
             padding-top: 0;
         }
+        .f1 {
```

Comment:
What is `f`?

Reply:
@kenjis  its font

Reply:
It seems we don't use names like `f1`.
Can you change to meaningful names?
`font-something`?

Reply:
ok @kenjis  i'll change it